incubator/hnc/README.md

```diff
@@ -46,10 +46,8 @@ kubectl apply -f https://github.com/kubernetes-sigs/multi-tenancy/releases/downl
 
 # Download kubectl plugin (Linux only) - will move to Krew soon
 PLUGIN_DIR=<directory where you keep your plugins - just has to be on your PATH>
-curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hierarchical_namespaces -o ${PLUGIN_DIR}/kubectl-hierarchical_namespaces
-chmod +x ${PLUGIN_DIR}/kubectl-hierarchical_namespaces
-# If desired to make 'kubectl hns' work:
-ln -s ${PLUGIN_DIR}/kubectl-hierarchical_namespaces ${PLUGIN_DIR}/kubectl-hns
+curl -L https://github.com/kubernetes-sigs/multi-tenancy/releases/download/hnc-${HNC_VERSION}/kubectl-hns -o ${PLUGIN_DIR}/kubectl-hns
+chmod +x ${PLUGIN_DIR}/kubectl-hns
```

Comment:
For v0.2, we still need the old instructions. Maybe add a v0.2 and v0.3 version (here and in the rest of the file)?

Reply:
I've made the updates. I personally think that it's confusing to layer in additional messages for 0.2.0 and before and 0.3.0 and later as it makes it increases complexity for things to pay attention to as a new person consuming this plugin. After all, history is available if they look at the source for v0.2.0. 

What do you think if we threw a banner at the top that links to 0.2.0 README.md so that those instructions can be easily seen?

Reply:
The only problem is that v0.3 isn't out yet. Maybe it's fine to have a warning banner; once we've release v0.3 it'll be fine to remove it.